fleet/lib/prometheus-alertrules/rules/ceph.yaml

```diff
@@ -0,0 +1,164 @@
+groups:
+  - name: ceph.rules
+    rules:
+      - alert: CephQuotaFillingUp
+        annotations:
+          summary: The Ceph pool quota in cluster {{ $labels.prom_cluster }} is almost full
+          description: |
+            Ceph pool id {{ $labels.pool_id }} on {{ $labels.prom_cluster }}/ {{
+            $labels.namespace }}/{{ $labels.pod }} is at {{ $value }}%. Please
+            keep in mind that ceph pools reaching 100% is dangerous.
+        labels:
+          secverity: warning
+          receivers: ",slack,"
+        expr: |
+          (ceph_pool_stored/ceph_pool_quota_bytes > 0.75 and ceph_pool_quota_bytes != 0)*100
+      - alert: CephQuotaFillingUp
+        annotations:
+          summary: The Ceph pool quota is almost full
+          description: |
+            Ceph pool id {{ $labels.pool_id }} on {{ $labels.prom_cluster }}/ {{
+            $labels.namespace }}/{{ $labels.pod }} is at {{ $value }}%. Please
+            keep in mind that ceph pools reaching 100% is dangerous.
+        labels:
+          secverity: critical
+          receivers: ",slack,"
+        expr: |
+          (ceph_pool_stored/ceph_pool_quota_bytes > 0.9 and ceph_pool_quota_bytes != 0)*100
+      - alert: CephTargetDown
+        expr: up{job=".*ceph.*"} == 0
```

Comment:
The wildcard matching does not seem to be working with prometheus on ayekan.  

Demonstration that there are `up` metrics with a label that includes `ceph`:
![image](https://github.com/user-attachments/assets/b82fe25f-9f53-4a54-90bb-74b2f5eb2bbe)

However, the expr in this rule doesn't match anything:

![image](https://github.com/user-attachments/assets/7d712ba1-b535-441e-829f-ac191ffe29e3)